RecoTracker/MeasurementDet/plugins/Chi2ChargeMeasurementEstimatorESProducer.cc

```diff
@@ -102,32 +103,36 @@ namespace {
 
 class  Chi2ChargeMeasurementEstimatorESProducer: public edm::ESProducer{
  public:
+  // static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
   Chi2ChargeMeasurementEstimatorESProducer(const edm::ParameterSet & p);
   virtual ~Chi2ChargeMeasurementEstimatorESProducer(); 
   boost::shared_ptr<Chi2MeasurementEstimatorBase> produce(const TrackingComponentsRecord &);
  private:
-  boost::shared_ptr<Chi2MeasurementEstimatorBase> _estimator;
+  boost::shared_ptr<Chi2MeasurementEstimatorBase> m_estimator;
+  const edm::ParameterSet m_pset;
+};
 
-  double maxChi2_;
-  double nSigma_;
-  float minGoodPixelCharge_; 
-  float minGoodStripCharge_;
-  float pTChargeCutThreshold_;
+/*  need support for RefToPSet
```

Comment:
is this support on the way?
If it's expected some time soon, it makes sense to keep this commented out code

@Dr15Jones